pandas/_libs/tslib.pyx

```diff
@@ -1917,7 +1927,7 @@ class Timedelta(_Timedelta):
     Parameters
     ----------
     value : Timedelta, timedelta, np.timedelta64, string, or integer
-    unit : string, [D,h,m,s,ms,us,ns]
+    unit : string, ``[D,h,m,s,ms,us,ns]``
```

Comment:
is this one needed to suppress a warning? 
I think we typically do not quote possible values in this part of the docstring (if we quote, I would rather quote them as strings to make it clear that is the option)

Reply:
Yeah... I *think* that caused a warning, though I can double check.

Reply:
Turns out it wasn't necessary. I've changed it to be the correct `{'ns', 'us', ...}` format.